openwisp_users/tests/test_models.py

```diff
@@ -333,3 +333,15 @@ def test_organization_user_string_representation(self):
 
         with self.subTest('Test user first and last names are empty'):
             self.assertEqual(str(org_user), f'{user.username} ({org.name})')
+
+    def test_add_user_method(self):
+        """
+        test adding user does not throw `INTEGRITY` error due to double owner creation
+        logic in `add_user` method and a signal trigger.
```

Comment:
Please remove this docstring as it's a redundant with the docstring of the `add_user` method, we do not need it (we can track the reason for this change by looking at the commit history and looking at this PR).